app/src/components/TaskAddForm.js

```diff
@@ -0,0 +1,130 @@
+import React, { useEffect, useState } from "react";
+
+import * as dbRequest from "../dbRequest";
+
+//Empty add form
+const initialState = {
+  summary: "",
+  description: "",
+  estTime: "",
+  dueDate: new Date(),
+};
+
+function reducer(state, action) {
+  switch (action.type) {
+    case "editSummary":
+      return { ...state, summary: action.value };
+
+    case "editDescription":
+      return { ...state, description: action.value };
+
+    case "editEstTime":
+      return { ...state, estTime: action.value };
+
+    case "editDueDate":
+      return { ...state, dueDate: action.value };
+
+    case "wipe":
+      return { ...initialState };
+    default:
+      return state;
+  }
+}
+
+//ADD FORM, CHILD OF EVENTBOARD
+function TaskAddForm({ setTasksList, timedownAccount }) {
+  const [state, dispatch] = React.useReducer(reducer, initialState);
+  const onSubmitForm = async (e) => {
+    e.preventDefault();
+    try {
+      const body = state;
+      console.log("Attempting to post...", JSON.stringify(body)); // TEST
+
+      const response = await dbRequest.addTask(
+        timedownAccount.id,
+        state.dueDate,
+        state.estTime,
+        state.summary,
+        state.description,
+      );
+
+      const update = JSON.stringify(response);
+      window.alert("Task submitted!");
+
+      const data = JSON.parse(update);
+
+      setTasksList(data);
+
+      dispatch({ type: "wipe", value: { initialState } });
```

Comment:
I see you're passing `{ initialState }` to `dispatch`, but it looks like the `wipe` action can already access `initialState` based on the scope that it's in.  Can you get away with just `dispatch({ type: "wipe"})` here instead?